NuGetPackages/MonoGame.Framework.MacOS.nuspec

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
+    <metadata>
+        <id>MonoGame.Framework.MacOS</id>
+        <version>0.0.0.0</version>
+        <title>MonoGame.Framework.MacOS</title>
+        <authors>MonoGameTeam</authors>
+        <owners>MonoGameTeam</owners>
+        <licenseUrl>https://github.com/mono/MonoGame/blob/develop/LICENSE.txt#L2</licenseUrl>
+        <projectUrl>http://monogame.net/</projectUrl>
+        <iconUrl>https://pbs.twimg.com/profile_images/487954549441691649/O3KsHAsb_400x400.png</iconUrl>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <description>MonoGame is an open source implementation of the Microsoft XNA 4.x Framework.
+      The goal is to make it easy for XNA developers to create cross-platform games with extremely high code reuse. We currently support iOS, Android, Windows (both OpenGL and DirectX), Mac OS X, Linux, Windows 8 Store, Windows Phone 8, PlayStation Mobile, and the OUYA console.</description>
+        <summary>MonoGame is an open source cross platform implementation of Microsoft's XNA 4.x Framework</summary>
+        <releaseNotes>Updated dev release - 10th December 2014 
+		Added support for Windows Phone 8.1 and Universal projects</releaseNotes>
+        <copyright>Copyright  2015</copyright>
+        <language>en-US</language>
+    </metadata>
+    <files>
+        <file src="MacOS\MonoGame.Framework.dll" target="lib\net40\MonoGame.Framework.dll" />
```

Comment:
> The core issue (as I understood it) is that you need to copy them from the mac build host to the windows host

Yep... i'm doing that now and placing them in the same location as if they were built on the Windows box.